install/kubernetes/cilium/values.yaml

```diff
@@ -0,0 +1,2041 @@
+# File generated by install/kubernetes/Makefile; DO NOT EDIT.
+# This file is based on install/kubernetes/cilium/values.yaml.tmpl.
```

Comment:
How was this file generated? Helm?

Reply:
It's generated using `envsubst`. Maybe I could have described this better in the PR description. Broadly what this PR does is it adds a bunch of exported variables in `install/kubernetes/Makefile.values`, then runs envsubst which takes in the `values.yaml.tmpl` on stdin plus the environment variables, then substitutes all of the variables in the template and writes out to this file.

Reply:
^ I expanded the PR description to help explain this, hopefully that makes sense now.